.github/auto-assign.yml

```diff
@@ -0,0 +1,29 @@
+# Set to true to add reviewers to issues/PRs
+addReviewers: true
+
+# Set to true to add assignees to issues/PRs
+addAssignees: true
+
+# Set to 'author' to add issue's/PR's author as a assignee
+addAssignees: author
```

Comment:
Imagine linus will come and create issue. And he will be assigned to it

Let's disable it

Reply:
yep